modal/mount.py

```diff
@@ -324,6 +324,9 @@ def add_local_dir(
         condition: Optional[Callable[[str], bool]] = None,
         # add files from subdirectories as well
         recursive: bool = True,
+        # Predicate filter function for file selection, which should accept a filepath and return `True` for inclusion.
```

Comment:
Is the explanation backwards now that we consider this as exclusive rather than inclusive?

Reply:
(Also I guess it's missing an explanation for the list of strings spelling).

Reply:
Yeah old explanation